datahub-web-react/src/app/ingest/source/IngestionSourceList.tsx

```diff
@@ -266,7 +266,7 @@ export const IngestionSourceList = () => {
                 message.success({ content: 'Removed ingestion source.', duration: 2 });
                 const newRemovedUrns = [...removedUrns, urn];
                 setRemovedUrns(newRemovedUrns);
-                setTimeout(function () {
+                setTimeout(() => {
```

Comment:
Not sure why these changes are strictly necessary, but we will leave them

Reply:
Oh, I forgot to mention it: ESLint reported it as an unexpected unnamed function...everywhere else in the code an arrow function was used, so I changed it.